File: src/Test/TCOFFICheatcode.php
<?php
$exports['runTCOFFICheatcode'] = function($limit) {
    $n = (int)$limit;
    $acc = 0;
    $i = $n;
    while ($i > 0) {
        $acc += $i;
        $i--;
    }
    return $acc;
};
return $exports;

File: src/Test/TCOFFICheatcode.rs
pub fn Test_TCOFFICheatcode_runTCOFFICheatcode(mut limit: i64) -> i64 {
    let mut acc = 0;
    let mut n = limit;
    while n > 0 {
        acc += n % 3;
        n -= 1;
    }
    acc
}
